addons/etcd/scripts/role-probe-script.sh

```diff
@@ -0,0 +1,19 @@
+tlsDir=$TLS_DIR
+status=""
+
+if [ -d $tlsDir ]; then
+  status=$(etcdctl --endpoints=127.0.0.1:2379 --cacert=${tlsDir}/ca.crt --cert=${tlsDir}/tls.crt --key=${tlsDir}/tls.key endpoint status -w simple --command-timeout=300ms --dial-timeout=100m)
+else
+  status=$(etcdctl --endpoints=127.0.0.1:2379 endpoint status -w simple --command-timeout=300ms --dial-timeout=100m)
+fi
```

Comment:
it's better to write a common.sh that do a wrapper around the etcdctl command

Reply:
sure!